clustering/marshalling/protostream/src/main/resources/org.wildfly.clustering.marshalling.protostream.proto

```diff
@@ -0,0 +1,29 @@
+package org.wildfly.clustering.marshalling.protostream;
+
+import "java.lang.proto";
+
+/**
+ * @TypeId(0)
```

Comment:
@pferraro Should we be using these IDs or is the Javadoc not specific enough when ti says 'globally'? Looking at some of the documentation:

> This numeric identifier must be globally unique so it can be used to identify the type instead of the fully qualified name.

> Values in the range 0..65535 (inclusive) are reserved for internal use by Protostream and other projects from the Infinispan organisation and should not be used by application developers.

Reply:
We do not reuse any of Infinispan's types, thus the ID space is our own.